src/rsz/src/Resizer.i

```diff
@@ -457,6 +457,20 @@ set_dont_touch_instance(Instance *inst,
   resizer->setDontTouch(inst, dont_touch);
 }
 
+void
+report_dont_use()
+{
+  Resizer *resizer = getResizer();
+  resizer->reportDontUse();
+}
+
+void
+report_dont_touch()
+{
+  Resizer *resizer = getResizer();
```

Comment:
I think this is missing `ensureLinked();` and that's the reason for the CI failure (it crashes in `commands_without_load.tcl`)

Reply:
thanks